packages/web3-providers-ws/test/unit/constructor.test.ts

```diff
@@ -0,0 +1,20 @@
+// import Web3ProviderBase from '../../src/index'
+// import {ProviderOptions} from '../../types'
+
+describe('constructs a PLACEHOLDER instance with expected properties', () => {
```

Comment:
You can use `it.todo` to add any placeholder. 

https://jestjs.io/docs/api#testtodoname